pkg/skaffold/update/update.go

```diff
@@ -31,20 +32,32 @@ import (
 // Fot testing
 var (
 	GetLatestAndCurrentVersion = getLatestAndCurrentVersion
+	readConfig                 = config.ReadConfigFileNoCache
```

Comment:
ReadConfigFileNoCache is only for testing - use `ReadConfigFile` instead

Reply:
I think there was a race detected via using ReadConfigFile. The reason being update check happens in background and both the background and foreground skaffold end up writing to the cache.
I will check again.

Reply:
This is no longer true. I think it was due to `SetUpFlags`.